superglue/package.json

```diff
@@ -56,6 +56,7 @@
     "redux-thunk": ">=2.3"
   },
   "dependencies": {
+    "@types/url-parse": "^1.4.11",
```

Comment:
How do you feel about placing this into devDependencies? I don't plan to export types until superglue has types to export. So i prefer a smaller step.

Reply:
In this file, could we modify the scripts to something like the below:

```diff
-    "build:index": "babel lib -d dist",
-    "build": "npm run clean && npm run build:index && npm run copy:package && npm run copy:readme",
+    "build": "npm run clean && npm run build:js && npm run copy:package && npm run copy:readme",
+    "build:js": "babel lib --out-dir dist --extensions \".ts,.tsx\" --source-maps inline",
+    "type-check": "tsc --noEmit",
+    "type-check:watch": "npm run type-check -- --watch",
```

This goes with the theme of just setting up typescript on the newly created `ts` files, while allowing it to build despite the errors we see.

Reply:
You'll also need to add `@babel/preset-typescript`, and make a modification to tsconfig. Something like

```
module.exports = {
  plugins: [
    ["transform-react-remove-prop-types", {removeImport: true}]
  ],
  presets: [
    [
      "@babel/preset-env",
      {
        useBuiltIns: "entry",
        corejs: "2",
        loose: true,
      }
    ],
    [
      "@babel/preset-react"
    ],
    [
      "@babel/preset-typescript"
    ]
  ],
};

```